apisix/plugins/ai.lua

```diff
@@ -116,7 +117,21 @@ end
 
 
 local pool_opt
-local function ai_balancer_run(route)
+local function ai_http_balancer_phase()
+    local api_ctx = ngx.ctx.api_ctx
+    if not api_ctx then
+        core.log.error("invalid api_ctx")
+        return core.response.exit(500)
+    end
+
+    if is_apisix_or then
+        local ok, err = response.skip_body_filter_by_lua()
```

Comment:
No skip_header_filter_by_lua?

Reply:
cancel skip_header_filter_by_lua, because we add `APISIX/2.99`in the header_filterphase, and `enable_server_tokens = false` cannot be used as a flag to skip header_filter_by_lua because it only removes the version in server token, not delete the server token unless we add another switch for removing the server token completely.